### ISSUE TITLE
Blank header on jitsi_meet screen

### ISSUE THREAD

**Issue by eliezerfaizal** [gunschu/jitsi_meet#10]
I'm running jitsi_meet plugin 0.1.4 with Flutter v1.12.13+hotfix.9 and Xcode 11.3.1.
Everything works well but there is a blank grey header on the screen. Is it a bug or I miss some configuration? thanks
![Uploading Simulator Screen Shot - iPhone 11 Pro Max - 2020-04-09 at 01.26.33.png…]()

**Comment by buzzwin** [comment]
That’s just how it’s coded for now ... we have a few ideas on how to enhance the experience ... did you have any suggestions ? 


Sent from Yahoo Mail for iPhone


On Wednesday, April 8, 2020, 1:31 PM, eliezerfaizal <notifications@github.com> wrote:




I'm running jitsi_meet plugin with Flutter v1.12.13+hotfix.9 and Xcode 11.3.1.
Everything works well but there is a blank grey header on the screen. Is it a bug or I miss some configuration? thanks

**Comment by rheenen** [comment]
First of all: Thanks for this plugin!

I also found that a blank grey header is shown at the top of a meeting, but in the sourcecode of the plugin I just cannot find where this header is 'inserted'. Tried standard Jitsi Meet examples in Swift and ObjectiveC and they do not show this grey header at all.

I fixed it by disabling the use of a UINavigationbar alltogether. This is also featured in DreamPowder's clone...

I changed the code in ios/Classes/SwiftJitsiMeetPlugin to contain this code at the bottom of the handle() method:

```
           self.uiVC.modalPresentationStyle = .fullScreen
            self.uiVC.present(jitsiViewController!, animated: true, completion: nil)
```
